bin/tessel-2.js

```diff
@@ -23,18 +23,21 @@ function closeSuccessfulCommand() {
   process.exit(0);
 }
 
-function closeFailedCommand(err) {
-  // If the returned value is an error
+// Allow options to be partially applied
+function closeFailedCommand(opts, err) {
```

Comment:
@MrNice what's the rationale modifying this function? You could previouslyt report an error if you pass it an error or you can have it warn the user if it's just a string